### ISSUE TITLE
Update the Database schema with missing fields

### ISSUE THREAD

**Issue by delageniere** [ispyb/ISPyB#310]
The ccAno and sigAno fields in AutoProcScalingStatistics. They were added in 2016_12_06_Added_ccAno_sigAno.sql but are not present in .mdb file.

Check if there are other fields, then update the .mdb and  the .pdf present on:
https://github.com/ispyb/ISPyB/tree/master/documentation/database

**Comment by delageniere** [comment]
There has been a problem of updating : the correct schema was ISPyB_dataModel_5.mwb, but in the 5.5.0 there are a lot of missing tables/fields.

It seems necessary to restart from the ISPyB_dataModel_5.mwb adding the fileds specific to 5.5.0

**Comment by antolinos** [comment]
We should probably just update one scheme with no version (ispyb.mwb) and then remove the rest of them.

**Comment by delageniere** [comment]
Yes, I agree, that makes sense and would avoid this type of problem.
